tests/unit/commands/test_remote.py

```diff
@@ -774,7 +774,7 @@ def test_monitor_build(mocker, emitter, snapcraft_yaml, base, fake_services):
     mock_fetch_artifacts.assert_called_once()
     mock_cleanup.assert_called_once()
 
-    emitter.assert_progress("Stopped: amd64")
```

Comment:
Although it's not necessarily a realistic thing that would happen, could you add another status before `SUCCESS` to ensure that `Stopped` still appears correctly?

Reply:
added!